Project/Src/Gui/TextView.cs

```diff
@@ -576,11 +578,11 @@ public WordFontPair(string word, Font font)
                 this.font = font;
             }
 
-            public override bool Equals(object obj)
+            public override bool Equals(object? obj)
             {
-                var myWordFontPair = (WordFontPair)obj;
```

Comment:
This would have thrown before if cast not OK, but a compare should not throw...
Reading this a couple of times to understand the old code, the new is better.